.travis.yml

```diff
@@ -95,8 +95,10 @@ jobs:
         - export CONDA_PATH="\$env:Path = ';C:\miniconda3;C:\miniconda3\Library\mingw-w64\bin;C:\miniconda3\Library\usr\bin;C:\miniconda3\Library\bin;C:\miniconda3\Scripts;C:\miniconda3\bin;C:\miniconda3\condabin;C:\iverilog\bin;' + \$env:Path"
         - powershell "$CONDA_PATH; conda install --yes -c msys2 m2-base m2-make m2w64-toolchain libpython"
         - powershell "$CONDA_PATH; conda install --yes virtualenv"
+        - powershell "$CONDA_PATH; conda update --yes pip"
+        - powershell "$CONDA_PATH; conda info; conda list --show-channel-urls"
```

Comment:
I'm guessing we are leaving this in to help diagnose future issues?

Reply:
It's generally a useful thing to have.

Reply:
Yes, it would have helped to diagnose this issue quicker and it doesn't hurt, so I'm leaving it in for the next time.